### DISCUSSION TITLE
Online Magick Studio

### DISCUSSION THREAD

**Discussion by wlc77** [ImageMagick/ImageMagick#2737] [General]
Sorry if this is the wrong place to post this but the online studio here: https://magickstudio.imagemagick.org/scripts/MagickStudio.cgi in annotate is missing a lot of fonts that it used to have a week or two ago. Including my favorite font I had used for years there. Many thanks to anyone who us able to get them back.

**Comment by urban-warrior** [comment]
Try now.  We added additional fonts.

**Reply by wlc77** [reply]
Thanks a lot i appreciate your help. I see many more added but still looks like my favorite (Signet Roundhand ATT Italic) is still missing unless i messed it somehow.
